lib/spdy-transport/connection.js

```diff
@@ -614,6 +652,7 @@ Connection.prototype._removeStream = function _removeStream(stream) {
 
   state.debug('id=0 remove stream=%d', stream.id);
   delete state.stream.map[stream.id];
+  state.stream.map[stream.id] = null;
```

Comment:
This is potentially very bad. It will lead to the hashmap that is going to grow with every closed request. Please consider removing this line, and checking against `undefined` everywhere else.